core/src/main/scala/chisel3/internal/firrtl/Converter.scala

```diff
@@ -45,6 +45,10 @@ private[chisel3] object Converter {
     reportInternalError(s"Trying to convert a cloned IO of $mod inside of $mod itself$loc!")
   }
 
+  // Create a map from base module to the component.
+  def getIdToComponent(components: Seq[Component]): VectorMap[BaseModule, Component] =
+    VectorMap[BaseModule, Component](components.map(c => c.id -> c): _*)
```

Comment:
```suggestion
    VectorMap(components.map(c => c.id -> c): _*)
```
This is a tiny nit, but is the duplication of the type parameters necessary?